### ISSUE TITLE
wrong calculation of point-plane distance

### ISSUE THREAD

**Issue by mcsoini** [sympy/sympy#18027]
I hope an an SO link suffices as a bug report: https://stackoverflow.com/questions/59245131/point-plane-distance-in-sympy/59245646

`sympy.geometry.plane` calculates the wrong distance between points an planes due to an erroneous calculation of the plane equation's coefficients. This can be fixed by simplifying the plane equation.

I can submit a PR if required.

**Comment by mcsoini** [comment]
... though it would likely be better to
* simplify the plane equation upon initialization
* or fix the calculation of the coefficients

**Comment by oscarbenjamin** [comment]
I think this is already fixed since the 1.4 release. On master I get:
```julia
In [1]: from sympy import *
   ...:  
   ...: a, b, c, d = symbols('a b c d') 
   ...: x1, y1, z1 = symbols('x_1 y_1 z_1') 
   ...:  
   ...: p1 = Point(x1, y1, z1) 
   ...: plane1 = Plane(Point(0, 0, -d/c), normal_vector=(a, b, c)) 
   ...:  
   ...: plane1.distance(p1)
Out[1]:
│                                              ⎛     d⎞   │
│                                            c⋅⎜z₁ + ─⎟   │
│       a⋅x₁                b⋅y₁               ⎝     c⎠   │
│───────────────── + ───────────────── + ─────────────────│
│   ______________      ______________      ______________│
│  ╱  2    2    2      ╱  2    2    2      ╱  2    2    2 │
│╲╱  a  + b  + c     ╲╱  a  + b  + c     ╲╱  a  + b  + c  │
```

**Comment by mcsoini** [comment]
Ok, I see this method was rewritten from scratch on master. Good to know.
The new version is not included in 1.4 though.
